### ISSUE TITLE
2.3.0 prints -hosts autofs entry

### ISSUE THREAD

**Issue by daniejstriata** [Canop/lfs#49]
Version 2.3.0 prints this autofs entry:
![image](https://user-images.githubusercontent.com/13314239/156140764-88034bb7-0fc4-4926-b39f-f2bc50d049a6.png)
which I believe should not be part of the defaults.

**Comment by Canop** [comment]
You're right. That was a side effect of whitelisting remote filesystems.

Just ot be sure of some things, can you please share the entry for this mount from `lfs -j` ?

**Comment by daniejstriata** [comment]
```
[
  {
    "bound": false,
    "dev": {
      "major": 8,
      "minor": 32
    },
    "disk": {
      "crypted": false,
      "ram": false,
      "removable": false,
      "rotational": true,
      "type": "HDD"
    },
    "fs": "/dev/sdc",
    "fs-label": "root",
    "fs-type": "ext4",
    "id": 21,
    "mount-point": "/",
    "remote": false,
    "stats": {
      "available": "5.2Gi",
      "bavail": 1353212,
      "bfree": 1536712,
      "blocks": 3579660,
      "bsize": 4096,
      "inodes": {
        "avail": 788801,
        "files": 917504,
        "free": 788801,
        "used-percent": "14%"
      },
      "size": "14Gi",
      "used": "8.5Gi",
      "used-percent": "62%"
    }
  },
  {
    "bound": false,
    "dev": {
      "major": 8,
      "minor": 1
    },
    "disk": {
      "crypted": false,
      "ram": false,
      "removable": false,
      "rotational": true,
      "type": "HDD"
    },
    "fs": "/dev/sda1",
    "fs-label": "boot",
    "fs-type": "ext2",
    "id": 23,
    "mount-point": "/boot",
    "remote": false,
    "stats": {
      "available": "383Mi",
      "bavail": 391781,
      "bfree": 417944,
      "blocks": 506724,
      "bsize": 1024,
      "inodes": {
        "avail": 131020,
        "files": 131072,
        "free": 131020,
        "used-percent": "0%"
      },
      "size": "495Mi",
      "used": "112Mi",
      "used-percent": "23%"
    }
  },
  {
    "bound": false,
    "dev": {
      "major": 253,
      "minor": 1
    },
    "disk": {
      "crypted": false,
      "ram": false,
      "removable": false,
      "rotational": true,
      "type": "LVM"
    },
    "fs": "/dev/mapper/vgroup-LVmysql",
    "fs-label": null,
    "fs-type": "ext4",
    "id": 24,
    "mount-point": "/var/lib/mysql",
    "remote": false,
    "stats": {
      "available": "172Gi",
      "bavail": 45142232,
      "bfree": 49453544,
      "blocks": 84859386,
      "bsize": 4096,
      "inodes": {
        "avail": 21561197,
        "files": 21561344,
        "free": 21561197,
        "used-percent": "0%"
      },
      "size": "324Gi",
      "used": "152Gi",
      "used-percent": "47%"
    }
  },
  {
    "bound": false,
    "dev": {
      "major": 253,
      "minor": 2
    },
    "disk": {
      "crypted": false,
      "ram": false,
      "removable": false,
      "rotational": true,
      "type": "LVM"
    },
    "fs": "/dev/mapper/vgroup-LVvarlocal",
    "fs-label": null,
    "fs-type": "ext4",
    "id": 25,
    "mount-point": "/var/app1-local",
    "remote": false,
    "stats": {
      "available": "186Gi",
      "bavail": 48659593,
      "bfree": 51667875,
      "blocks": 59225268,
      "bsize": 4096,
      "inodes": {
        "avail": 14895374,
        "files": 15056896,
        "free": 14895374,
        "used-percent": "1%"
      },
      "size": "226Gi",
      "used": "40Gi",
      "used-percent": "18%"
    }
  },
  {
    "bound": false,
    "dev": {
      "major": 253,
      "minor": 3
    },
    "disk": {
      "crypted": false,
      "ram": false,
      "removable": false,
      "rotational": true,
      "type": "LVM"
    },
    "fs": "/dev/mapper/vgroup-LVvarapp1",
    "fs-label": null,
    "fs-type": "ext4",
    "id": 26,
    "mount-point": "/var/app1",
    "remote": false,
    "stats": {
      "available": "112Gi",
      "bavail": 29324741,
      "bfree": 31028049,
      "blocks": 33508170,
      "bsize": 4096,
      "inodes": {
        "avail": 8472749,
        "files": 8519680,
        "free": 8472749,
        "used-percent": "1%"
      },
      "size": "128Gi",
      "used": "16Gi",
      "used-percent": "12%"
    }
  },
  {
    "bound": false,
    "dev": {
      "major": 253,
      "minor": 0
    },
    "disk": {
      "crypted": false,
      "ram": false,
      "removable": false,
      "rotational": true,
      "type": "LVM"
    },
    "fs": "/dev/mapper/vgroup-LVusrapp1",
    "fs-label": null,
    "fs-type": "ext4",
    "id": 27,
    "mount-point": "/usr/local/app1",
    "remote": false,
    "stats": {
      "available": "7.1Gi",
      "bavail": 1870290,
      "bfree": 2001362,
      "blocks": 2547534,
      "bsize": 4096,
      "inodes": {
        "avail": 640350,
        "files": 655360,
        "free": 640350,
        "used-percent": "2%"
      },
      "size": "9.7Gi",
      "used": "2.6Gi",
      "used-percent": "27%"
    }
  },
  {
    "bound": false,
    "dev": {
      "major": 0,
      "minor": 20
    },
    "disk": null,
    "fs": "-hosts",
    "fs-label": null,
    "fs-type": "autofs",
    "id": 30,
    "mount-point": "/net",
    "remote": true,
    "stats": {
      "available": "0",
      "bavail": 0,
      "bfree": 0,
      "blocks": 0,
      "bsize": 4096,
      "inodes": null,
      "size": "0",
      "used": "0",
      "used-percent": "0%"
    }
  }
]

```

**Comment by Canop** [comment]
I'll start by not considering stats with a total count of blocks 0 as being valid...
